ext/rj_schema/rj_schema.cpp

```diff
@@ -23,6 +23,8 @@
 #include <ruby.h>
 #include <ruby/version.h>
 
+#include <stdio.h>
```

Comment:
Preferably include `<cstdio>` instead and then prefix the file functions with `std::`